packages/core/contracts/common/implementation/Stakeable.sol

```diff
@@ -0,0 +1,50 @@
+/**
+ * Stakeable contract.
+ */
+
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../../oracle/interfaces/StakerInterface.sol";
+import "./Withdrawable.sol";
+
+/**
+ * @title Base contract that extends the Withdrawable contract enabling a specific role to stake ERC20 tokens against the
+ * Voting contract. Voting contract is fed in as a param rather than fetched from the finder to enable upgradability.
+ */
+abstract contract Stakeable is Withdrawable {
```

Comment:
Just OOC, why did you choose to make this a separate contract rather than just adding these methods to the DesignatedVotingV2 contract?

Reply:
to follow the same pattern that the DesignatedVoting v1 had. can roll it all into one but by doing it this way it follows the same pattern.